2022q4/azure.adoc

```diff
@@ -0,0 +1,42 @@
+=== FreeBSD on Microsoft HyperV and Azure
+
+Links: +
+link:https://wiki.freebsd.org/MicrosoftAzure[Microsoft Azure article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/MicrosoftAzure[] +
+link:https://wiki.freebsd.org/HyperV[Microsoft HyperV article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/HyperV[]
+
+Contact: Microsoft FreeBSD Integration Services Team <bsdic@microsoft.com> +
+Contact: link:https://lists.FreeBSD.org/mailman/listinfo/freebsd-cloud[freebsd-cloud Mailing List] +
+Contact: The FreeBSD Azure Release Engineering Team <releng-azure@FreeBSD.org> +
+Contact: Wei Hu <whu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+
+In this quarter, the link:https://azuremarketplace.microsoft.com/marketplace/apps/thefreebsdfoundation.freebsd-12_4[12.4-RELEASE image] has been published to Azure Marketplace.
+
+Work in progress tasks:
+
+* Automating the image building and publishing process and merge to src/release/.
+* Building and publishing ZFS-based images to Azure Marketplace
+** All the requirted codes are merged to main branch, and can create ZFS-bared image by specifying `VMFS=zfs`.
+** Need to make the build process more automaitc and collaborating with release engineering to start generating snapshots.
+* Building and publishing Hyper-V gen2 VM images to Azure Marketplace
+** Blocked by https://bugs.freebsd.org/264267
+
+The above tasks are sponsored by The FreeBSD Foundation, with resources provided by Microsoft.
+
+Wei Hu and his colleagues in Microsoft are working on several tasks sponsored by Microsoft:
+
+* Fixing booting issue on Hyper-V gen2 VM in Azure
+** https://bugs.freebsd.org/264267
+* Porting Hyper-V guest support to aarch64
+** https://bugs.freebsd.org/266248
+** https://bugs.freebsd.org/267654
+
+Open tasks:
+
+* Update FreeBSD related doc at link:https://docs.microsoft.com[]
```

Comment:
`link:https://docs.microsoft.com[https://docs.microsoft.com]` (or more explicit link if there's a specific subdir for FreeBSD)